torchci/lib/rolloutUtils.ts

```diff
@@ -0,0 +1,3 @@
+export const POSSIBLE_USERS = ["swang392", "janeyx99", "atalman", "clee2000", "seemethere",
+    "huydhn", "zengk95", "linbinyu", "suo", "dagitses", "malfet", "mehtanirav", "kit1980",
```

Comment:
I don't think this is the rolloutUtils that currently exists. it's in lib/bot/rolloutUtils

Reply:
oh oops